upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -115,11 +122,14 @@ func (s *Route) CheckChanges(a, e, changes *Route) error {
 		if e.Instance != nil {
 			targetCount++
 		}
+		if e.NatGateway != nil {
+			targetCount++
+		}
 		if targetCount == 0 {
-			return fmt.Errorf("InternetGateway or Instance is required")
+			return fmt.Errorf("InternetGateway | Instance | NatGateway is required")
```

Comment:
This was inspired by good ol *nix services 

EG: `service myserviced start|stop|restart|re;pad`

Basically the way this **should** read is :

You need to define an InternetGateway, and Instance, or a NatGateway or this won't work